hooks/useNavigation.tsx

```diff
@@ -63,5 +63,10 @@ export const useNavigation = () => {
         context
       );
     },
+    newNavigationItems: (() => {
```

Comment:
Why removing Get Involved from the Nav Bar? I know it's not on the Figma but we should render the Navigation as it is.

We will in the future rearrange the items and rename them, so this method is not needed.